schematron/v2.2.0/v2-2-0_L200_Audit.csv

```diff
@@ -45,6 +45,8 @@
 "","","","","","","","","","//auc:HVACSystem[auc:LinkedPremises/auc:Section/auc:LinkedSectionID/@IDref = current()/@ID]","auc:Section[auc:SectionType='Space function'] must have a linked auc:HVACSystem","",""
 "","","","","","","","","","//auc:LightingSystem[auc:LinkedPremises/auc:Section/auc:LinkedSectionID/@IDref = current()/@ID]/auc:LampType","auc:Section[auc:SectionType='Space function'] must have a linked auc:LightingSystem with auc:LampType defined","",""
 "","","","","","","","","","//auc:LightingSystem[auc:LinkedPremises/auc:Section/auc:LinkedSectionID/@IDref = current()/@ID]/auc:LampType//auc:LampLabel","auc:Section[auc:SectionType='Space function'] must have a linked auc:LightingSystem with auc:LampLabel defined","WARNING",""
+"","","Scenarios","","","/auc:BuildingSync/auc:Facilities/auc:Facility/auc:Reports/auc:Report/auc:Scenarios/auc:Scenario","true","","","auc:LinkedPremises/auc:Building/auc:LinkedBuildingID","","",""
+"","","","","","","","","","//auc:Buildings/auc:Building[@ID = current()/auc:LinkedPremises/auc:Building/auc:LinkedBuildingID/@IDref]","Every auc:Scenario must be linked to an auc:Building through auc:LinkedPremises","",""
```

Comment:
@nllong please confirm.  We just added this requirement.  Question is, should this be more generic, i.e. `Each Scenario must be linked to a type of premise: Facility, Site, Building`.  At this point in time, we have pretty exclusively been using scenarios to represent different 'building configurations', i.e. whether it is the current building modeled, current building modeled + package of measures, etc.

Reply:
good for now.  Revisit maybe later.